fabric_cf/orchestrator/swagger_client/api/slices_api.py

```diff
@@ -246,43 +246,45 @@ def slices_delete_slice_id_delete_with_http_info(self, slice_id, **kwargs):  # n
             _request_timeout=params.get('_request_timeout'),
             collection_formats=collection_formats)
 
-    def slices_get(self, **kwargs):  # noqa: E501
+    def slices_get(self, state, **kwargs):  # noqa: E501
```

Comment:
This is more of a comment - what are the kwargs here - presumably you wanted state to be required, that's why it isn't part of kwargs?

Reply:
These modules are generated by swagger. I downloaded this code from python client using swagger definition from swagger hub.

Reply:
I c. All good then